### PR TITLE
Vega chart for Survey

### PR DESCRIPTION
**Pull request by crashGoBoom**
# Description

Hi @statik !
This PR will add the initial survey chart created with vega/vega-lite.

This will also bump js versions for:
vega-embed to v4.0.0
vega-lite to v3.3.0
vega to v5.4.0

### CODE REVIEW

File: webui/src/Visualization.elm [hunk 1]
```diff
@@ -1,64 +1,233 @@
...
             -- Do something here with some data eventually!
             getData
 
+        questions =
+            Zipper.toList model.questions
+
+        answerListing =
+            getAnswers questions
+
+        debugLog =
```
Comment (statik) [inline webui/src/Visualization.elm]: I think debugLog has to be removed for a production build.

File: webui/src/Page/Survey.elm [hunk 2]
```diff
@@ -1172,7 +1172,16 @@ viewRegistration model =
                 , input [ placeholder "Email Address", value model.emailAddress, onInput UpdateEmail ] []
                 , br [] []
                 , br [] []
-                , div [ class "vis", id "vis" ] []
+                , div [ class "vis", id "vis0" ] []
```
Comment (statik) [inline webui/src/Page/Survey.elm]: Now that we are back at a single spec, can this change be dropped?

File: webui/src/index.js [hunk 3]
```diff
@@ -42,12 +42,16 @@ elmApp.ports.saveSurveyState.subscribe(storedSurvey => {
...
 })
 
-let updateChart = function (spec) {
+let updateChart = function (specs) {
   //console.log("updateChart was called");
   window.requestAnimationFrame(() => {
-    var element = $('#vis');
-    if (element) {
-      vegaEmbed("#vis", spec, { actions: false }).catch(console.warn);
+    var element = $('#vis0');
```
Comment (statik) [inline webui/src/index.js]: now that we are back at a single spec, can this change be dropped?

File: webui/src/Visualization.elm [hunk 4]
```diff
@@ -1,64 +1,233 @@
...
+            Debug.log "List of answers for questions" answerListing
+
+        -- TODO
```
Comment (statik) [inline webui/src/Visualization.elm]: Are these comments still correct?

Comment (crashGoBoom) [reply webui/src/Visualization.elm]: I removed debugs and those old comments. Just now. Let me know what you think or if i missed anything.

**Comment by crashGoBoom** [comment]
> Looks awesome! Some minor changes needed like removing all `Debug.log calls` for the production build to pass.

Just pushed some more cleaning up changes. I will remove the WIP since i think it looks pretty clean now. Let me know if i missed anything. Thanks!

**Pull request merged**